pkg/starboard/config.go

```diff
@@ -356,6 +357,17 @@ func (c ConfigData) GetTrivyServerURL() (string, error) {
 	return c.getRequiredProperty(keyTrivyServerURL)
 }
 
+func (c ConfigData) GetTrivyInsecureRegistries() []string {
```

Comment:
I'd return a map[string]bool here so we don't duplicate the integration logic in plugin.go

Reply:
Also we should have a unit test for new methods

Reply:
updated to map[string]bool and added unit test.